design/product-design-list-all-accounts.md

```diff
@@ -0,0 +1,75 @@
+# LIST AND CREATE ACCOUNTS
+
+## List accounts
+
+Shows all accounts in system, has a button that allows to go to the account creation view.
+The name of each acount can be clicked and this will lead to a blank page (for the moment)
+
+## Create account
+
+Shows a form to enter the account information:
+
+- Account name: Must be unique among all other names on the system.
+- Person fullname: should be minimum 1 character long or maximun 255 characters.
+
+Finally when we press submit button, if the content provided in the form is valid, this will redirect to list accounts page, else will show an feedback error message showing form mistakes.
+
+## Wireframes
+
+Images (mocks)
+
+- List accounts page
+- Forms page
+- Forms page error
+
+## API contract
+
+### List accounts page
+
+| RES API spec  | content       |
+| ------------- |:-------------:|
+| Request headers     | Content-Type: application/json |
+| Request HTTP method | GET      |
+| Request URL       | /account/list      |
+| Request body       | ---      |
+| Response headers       | Content-Type: application/json     |
```

Comment:
I like this format. I'd keep it for everything but the request and response bodies, which quickly will become unreadable.